go.mod

```diff
@@ -140,7 +140,7 @@ require (
 	github.com/samuel/go-thrift v0.0.0-20140522043831-2187045faa54
 	github.com/sanathkr/yaml v1.0.1-0.20170819201035-0056894fa522 // indirect
 	github.com/shopspring/decimal v1.2.0
-	github.com/spf13/cobra v1.3.0
+	github.com/spf13/cobra v1.6.1
```

Comment:
The tests that are failing here are related to shell completion commands, and the build is passing reliably on main https://beats-ci.elastic.co/job/Beats/job/beats/job/main/

`github.com/spf13/cobra` is the package used to implement the Beats command line parsing, it is possible this upgrade triggered the failures. If you revert this to the previous version with `github.com/spf13/cobra@v1.3.0` do the test failures go away?

It might be faster to try this locally instead of in CI, run `mage unitTest` in the libbeat directory to see if you can reproduce the problem on your local machine first and then try the version change suggested above.

Reply:
I do specifically see some references to command completion changes in https://github.com/spf13/cobra/releases/tag/v1.6.0

Reply:
I tested locally with the change you suggested and the test that fails here, passed locally, but other tests that are OK here failed.

Reply:
@cmacknz the golang cobra library was the issue, thanks for looking into this!